pyhive/tests/test_sqlalchemy_hive.py

```diff
@@ -175,41 +197,39 @@ def test_lots_of_types(self, engine, connection):
             "a", 1, 1,
             0.1, 0.1, 0, 0, 0, "a",
             false, "a", "a",
-            0, %d, 123 + 2000
+            0, :big_number, 123 + 2000
         FROM default.one_row
-        """, big_number)
-        row = connection.execute(table.select()).fetchone()
-        self.assertEqual(row.hive_date, datetime.date(1970, 1, 1))
+        """), {"big_number": big_number})
+        row = connection.execute(text("select * from test_table")).fetchone()
+        self.assertEqual(row.hive_date, datetime.datetime(1970, 1, 1, 0, 0))
```

Comment:
Corrected this test case since it was already failing with SQLAlchemy 1.3 and 1.4.